package.json

```diff
@@ -365,6 +365,11 @@
           "default": "",
           "description": "Specifies platform used for multi-arch images.",
           "scope": "window"
+        },
+        "redHatDependencyAnalytics.recommendations.enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "Toggles recommending Red Hat repositories"
```

Comment:
Going to expand on this description I think. The PR as-is toggles all recommendations. Im not sure if we do any recommendations for non-redhat alternatives though, so its entirely possible this description is accurate